### DISCUSSION TITLE
Steps to deploy OC to distributed systems with docker

### DISCUSSION THREAD

**Discussion by briziomusic** [OrchardCMS/OrchardCore#11848] [Q&A]
Hello,
I'm looking for some documentation about how to deploy OrchardCore to distributed systems through docker containers.
For distributed systems i mean some cloud providers or systems like kubernets.
Thanks

**Comment by Skrypt** [comment]
Here are some basics; if working with source code.

https://github.com/OrchardCMS/OrchardCore/tree/main/src/docs/topics/docker

There is currently no specific documentation about using Kubernetes or "how to" documentation on how to deploy Docker containers.
This should generally work by using these systems' documentation.
